packages/theme/src/scale-large.ts

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import largeStyles from './scale-large.css.js';
+import { Theme } from './theme';
+
+Theme.registerThemeFragment('large', 'size', largeStyles);
```

Comment:
I've noticed there's a naming discrepancy between `size` and `scale` across this code... should we try to centralize on what's going on in Spectrum: e.g. `scale`?

Reply:
@cuberoot do you have any thoughts on this? It'd be an API breaking change, so we'd want to get to as soon as possible if we did.